subiquity/common/types.py

```diff
@@ -362,7 +362,7 @@ class WSLConfigurationBase:
 
 
 @attr.s(auto_attribs=True)
-class WSLConfigurationAdvanced:
+class WSLConfigurationAdvanced(WSLConfigurationBase):
```

Comment:
We removed than inheritance, didn’t we?

Reply:
I believe this comes from main. Fixed anyway.

Reply:
This comes from your first commit on this branch ee1976a10ebf92868d3a67707a4ae231eb032d78.
Please edit this commit and remove the inheritance from there.

Reply:
fixed